### ISSUE TITLE
Issue with Landscape view of Text - INSLabelPullToRefresh

### ISSUE THREAD

**Issue by PPacie** [inspace-io/INSPullToRefresh#38]
![img_3185-2](https://cloud.githubusercontent.com/assets/4922383/7483740/c6ec1b28-f357-11e4-8a16-814ecbca4116.PNG)

I couldn't fix it. Thank you very much.

**Comment by PPacie** [comment]
Found a hack, it works but it's not very nice. I mean, it works fine but it doesn't create the better ux when you trigger the pull to refresh and rotate the screen right after =)

- (void) willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration {
    [self.tableView ins_endPullToRefresh];
    [self.tableView ins_removePullToRefresh];
}
- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation {
 [self _createPullToRefresh];  
}

Where [self _createPullToRefresh] is a function that I created to include all the parameters [self.tableview ins_addPullToRefreshWithHeight:...], etc.

Please let me know if you are aware of any other better alternatives to it.

Cheers,
Pablo.

**Comment by m1entus** [comment]
Did you rey - (void)ins_setPullToRefreshEnabled:(BOOL)enabled; ?

**Comment by PPacie** [comment]
Yes, It doesn't work with the INSLabelPullToRefresh because of the frame of the label. This frame doesn't get updated after Enable/Disable with ins_setPullToRefreshEnabled.

**Comment by m1entus** [comment]
In INSLabelPullToRefresh, layoutSubviews set self.bounds = self.superView.bounds or add constraints to superview. It will work

**Comment by PPacie** [comment]
Thank you very much!
